### PR TITLE
Github actions: Move to the new GitHub container registry, which allo…

### PR DESCRIPTION
**Pull request by quartje**
…ws anonymous access to the containers

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -21,8 +21,8 @@ jobs:
       with:
         dockerfile: docker/php-fpm/Dockerfile
         username: ${{ github.actor }}
-        password: ${{ secrets.GITHUB_TOKEN }}
-        registry: docker.pkg.github.com
+        password: ${{ secrets.GH_CONTAINER_REGISTRY_PAT }}
```
Comment (MKodde) [inline .github/workflows/main.yml:24]: `PATH` ?

**Review by MKodde — COMMENTED** [review]
Might I've found a typo?

File: .github/workflows/main.yml (hunk 1 above)
Comment (quartje) [reply .github/workflows/main.yml:24]: PAT = Personal Access Token

Comment (MKodde) [reply .github/workflows/main.yml:24]: Ah, the more you know!

**Review by MKodde — APPROVED** [review]

**Pull request merged**